api/api.go

```diff
@@ -128,3 +131,28 @@ func (w Wrapper) UpdateServiceProvider(ctx echo.Context) error {
 	return ctx.JSON(200, spRequest)
 }
 
+func (w Wrapper) ConnectCustomer(ctx echo.Context, id string) error {
+	_, err := w.checkAuthorization(ctx)
+	if err != nil {
+		return err
+	}
+
+	nodeClient := v1.HTTPClient{
+		ServerAddress: w.NodeAddress,
+		Timeout:       1 * time.Second,
+	}
+
+	didDocument, err := nodeClient.Create()
+	if err != nil {
+		return err
+	}
+
+	did := didDocument.ID.String()
```

Comment:
when we store customers, do we allow 'connecting' them more than once?

Reply:
not via UI, but via backend, yes. connect is just generating a new DID